Installer/data/BrcmPatchRAM/BrcmBluetoothInjector.kext.MaxKernel.txt

```diff
@@ -0,0 +1 @@
+20.9.9
```

Comment:
Did you mean 21.9.99?

Reply:
Nope. This kext must not be used in Monterey(>=21.x.x). But is needed in Big sur (<= 20.9.9)

Reply:
MinKernel is the minimum version that does support it so it should be 21.0.0 right?

Reply:
Sorry, I don't quite follow you.

The fact is:
- BlueToolFixup is only needed in Monterey, so MinKernel for it is >=21.0.0
- BrcmBluetoothInjector or IntelBluetoothInjector must not be used in Monterey, so MaxKernel for them both <=20.99.99

Reply:
Sorry I didn’t realize they were two different files. Anyways can we do 20.99.99 on the off chance Apple releases 10 updates?

Reply:
Of course